polynote-frontend/polynote/codec.js

```diff
@@ -50,9 +50,9 @@ export class DataReader {
 
     // NOTE: JavaScript can't represent all 64-bit integers...
     readInt64() {
-        const hi = this.readInt32();
-        const lo = this.readInt32();
-        return (hi << 32) | lo;
+        const result = this.buffer.getBigInt64(this.offset);
```

Comment:
😲 is this a Chrome-specific thing?

Reply:
Chrome, FF, Opera looks like: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Browser_compatibility